web/src/layout/leftpanel/countrypanel/index.jsx

```diff
@@ -95,6 +99,24 @@ const CountryPanel = ({ electricityMixMode, isMobile, tableDisplayEmissions, zon
     return <Redirect to={parentPage} />;
   }
 
+  // Redirect to the parent page if the zone is subzone and country view is enabled
+  if (
+    zones[zoneId].geography.properties.isHighestGranularity === true &&
+    zones[zoneId].geography.properties.isAggregatedView === false &&
+    isAggregateEnabled === true
```

Comment:
It may be more subjective but I personally don't use this explicit boolean typing.

I think it can make sense in some cases without typescript to emphasise that we're working with a boolean but since these booleans are prefixed with 'is' I think it's more readable to just say


```
    zones[zoneId].geography.properties.isHighestGranularity &&
    !zones[zoneId].geography.properties.isAggregatedView &&
    isAggregateEnabled
```

If you wanted to you could destructure the booleans out before so you'd just say

```
    isHighestGranularity &&
    !isAggregatedView &&
    isAggregateEnabled
```

Reply:
Because this is an edge case, I wanted to make these cases as strict as possible. For example is isAggregatedView is undefined or null I wouldn't want this to execute

Reply:
Hmm okay i didn't read it as such. I guess I would usually have a guard clause specifically for those cases if I didn't want to execute.

Reply:
Yeah I think yours is more readable, hopefully we make it to the promised typed land :)